client/main.py

```diff
@@ -1,9 +1,11 @@
 import argparse
 import json
+import uvicorn
+
 
 import requests
 
-BASE_URL = "localhost:8000"
+BASE_URL = "http://localhost:8000/"
```

Comment:
This is not in align with the rest of the code, check strings where you're using this variable.

Also, personally I prefer to not adding the trailing slash, because in this case you string would be looking like this:
```python
f"{BASE_URL}path-to-your-endpoint"
```

Instead of like this:
```python
f"{BASE_URL}/path-to-your-endpoint"
```

Which is a little bit more pretty. But not a big deal, it's rather a matter of taste.